db/sqldb/portfolio.go

```diff
@@ -215,16 +232,22 @@ func rowToPortfolio(row rowScanner) (*pacta.Portfolio, error) {
 	if err := checkSizesEquivalent("initiatives", len(initiativesIDs), len(initiativesAddedByIDs), len(initiativesCreatedAts)); err != nil {
 		return nil, err
 	}
+	seenInitiativeIDs := make(map[pacta.InitiativeID]bool)
 	for i := range initiativesIDs {
 		if !initiativesIDs[i].Valid && !initiativesCreatedAts[i].Valid {
 			continue // skip nulls
 		}
-		if !groupsIDs[i].Valid {
-			return nil, fmt.Errorf("portfolio group membership ids must be non-null")
+		if !initiativesIDs[i].Valid {
+			return nil, fmt.Errorf("initiative ids must be non-null")
 		}
-		if !groupsCreatedAts[i].Valid {
-			return nil, fmt.Errorf("portfolio group membership createdAt must be non-null")
+		if !initiativesCreatedAts[i].Valid {
+			return nil, fmt.Errorf("initiative createdAt must be non-null")
+		}
+		id := pacta.InitiativeID(initiativesIDs[i].String)
+		if seenInitiativeIDs[id] {
+			continue // Skip duplicates, see MULTI_LEFT_JOIN note.
```

Comment:
I don't see `MULTI_LEFT_JOIN` occur anywhere else in the codebase.

Reply:
Excellent catch - this was a leftover of one of my "row munging" strategies from the previous comment which did not work. Removed.